src/PrestaShopBundle/Form/Admin/Sell/Product/SEO/RedirectOptionType.php

```diff
@@ -155,17 +170,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         parent::configureOptions($resolver);
-        $resolver->setDefaults([
-            'required' => false,
-            'label' => $this->trans('Redirection page', 'Admin.Catalog.Feature'),
-            'label_tag_name' => 'h2',
-            'label_help_box' => $this->trans('When your product is disabled, choose to which page you’d like to redirect the customers visiting its page by typing the product or category name.', 'Admin.Catalog.Help'),
-            'columns_number' => 2,
-            'row_attr' => [
-                'class' => 'redirect-option-widget',
-            ],
-            'alert_message' => $this->getRedirectionAlertMessages(),
-        ]);
+        $resolver
+            ->setDefaults([
+                'product_id' => null,
+                'required' => false,
+                'label' => $this->trans('Redirection page', 'Admin.Catalog.Feature'),
+                'label_tag_name' => 'h2',
+                'label_help_box' => $this->trans('When your product is disabled, choose to which page you’d like to redirect the customers visiting its page by typing the product or category name.', 'Admin.Catalog.Help'),
```

Comment:
I think we can remove this tooltip, it's redundant. :-) The label "Redirection page" and the help text "Redirection when offline" are already clear enough in my opinion.

Reply:
Hi, I'm not sure this feature is very technical Some merchant don't even know what a "redirection" means And this page already has a lot of different fields to setup, in this case a little extra info doesn't hurt IMHO Especially since it's in a tooltip so it doesn't bother the use of the page unless you put your mouse cursor on it.

See video https://watch.screencastify.com/v/EWaJZ5bf8cCOwWXz2G6s

Reply:
As discussed with @Julievrz This part can be improved (too many blocks and/or tooltip on this page). But this improvement will be done later globally on the whole page For now this PR can be merged as-is to avoid blocking coming development regarding pakcs